rllib/offline/estimators/fqe_torch_model.py

```diff
@@ -138,8 +144,10 @@ def train_q(self, batch: SampleBatch) -> TensorType:
                 state_keys = ["state_in_{}".format(i) for i in range(num_state_inputs)]
 
                 # Compute action_probs for next_obs as in FQE
-                all_actions = torch.zeros([minibatch.count, self.policy.action_space.n])
-                all_actions[:] = torch.arange(self.policy.action_space.n)
+                all_actions = torch.zeros(
```

Comment:
**question**: I don't understand the tensor operations here. What are you trying to do? Why did you create a torch.zeros()? And why are you doing .T afterwards? I have a feeling that this can be written in a better way but I don't understand it to make any recommendations.

Reply:
This was a quick but hacky way of computing the `next_action_probs` for all actions in the action space, then multiplying it with the next Q-values to get the expected value `V(s) = \sum_a \pi(a | s) * Q(s,a)`.

The transposes weren't really clear, so I've changed it to a for loop over the action space, same as in `estimate_v` in the same file.